seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/seatunnel.yaml

```diff
@@ -39,4 +39,4 @@ seatunnel:
       metric:
          enabled: false
       logs:
-         scheduled-deletion-enable: true
+         scheduled-deletion-enable: false
```

Comment:
why change this?

Reply:
If this set to true, it will start the log deletion thread to delete log files, first it will lookup the log folder.

But in CI, log4j is using console print, won't write log to file.  https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-e2e-common/src/test/resources/log4j2.properties#L25

So when this config enabled, it will get exception 
![image](https://github.com/user-attachments/assets/48cb3ffe-9d65-4b17-b65c-eb69545ca6be)

Reply:
cc @corgy-w

Reply:
This problem does exist. Is there any better way to avoid this problem

Reply:
Can we disable it in CI as we won't write log file during CI and there has no test using this config.

Reply:
Ok for me. The delete expired log file test case config is https://github.com/apache/seatunnel/blob/e22e07726e00531f48700265b55d3c3e40c06d88/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/cluster/seatunnel.yaml#L42.